controllers/controller.js

```diff
@@ -16,4 +16,17 @@ const listArticles = (request, response) => {
     })
 }
 
-module.exports = { listTopics, listArticles, handle404Errors }
\ No newline at end of file
+const findSpecificArticle = (request, response, next) => {
+  findArticleById(request.params.article_id)
+    .then((article) => {
+      response.status(200).send({ article: { article } });
```

Comment:
You may need to handle an error in your own way here, to differentiate a 400 from a 404